### ISSUE TITLE
Dying crashes the client

### ISSUE THREAD

**Issue by cyclowns** [space-wizards/space-station-14#2886]
To reproduce:
1) Die (running out of an airlock is a quick and easy way)

<details><summary>Exception details</summary>
<p>

```
Robust.Client.GameObjects.ComponentStateApplyException
  HResult=0x80131500
  Message=Failed to apply comp state: entity=Jadon Priebe (5204, HumanMob_Content), comp=Alerts
  Source=Robust.Client
  StackTrace:
   at Robust.Client.GameObjects.ClientEntityManager.HandleEntityState(IComponentManager compMan, IEntity entity, EntityState curState, EntityState nextState) in K:\github.com\cyclowns\space-station-14\RobustToolbox\Robust.Client\GameObjects\ClientEntityManager.cs:line 296
   at Robust.Client.GameObjects.ClientEntityManager.ApplyEntityStates(EntityState[] curEntStates, IEnumerable`1 deletions, EntityState[] nextEntStates) in K:\github.com\cyclowns\space-station-14\RobustToolbox\Robust.Client\GameObjects\ClientEntityManager.cs:line 98
   at Robust.Client.GameStates.ClientGameStateManager.ApplyGameState(GameState curState, GameState nextState) in K:\github.com\cyclowns\space-station-14\RobustToolbox\Robust.Client\GameStates\ClientGameStateManager.cs:line 381
   at Robust.Client.GameStates.ClientGameStateManager.ApplyGameState() in K:\github.com\cyclowns\space-station-14\RobustToolbox\Robust.Client\GameStates\ClientGameStateManager.cs:line 208
   at Robust.Client.GameController.Tick(FrameEventArgs frameEventArgs) in K:\github.com\cyclowns\space-station-14\RobustToolbox\Robust.Client\GameController.cs:line 277
   at Robust.Client.GameController.<MainLoop>b__66_0(Object sender, FrameEventArgs args) in K:\github.com\cyclowns\space-station-14\RobustToolbox\Robust.Client\GameController\GameController.Standalone.cs:line 85
   at Robust.Shared.Timing.GameLoop.Run() in K:\github.com\cyclowns\space-station-14\RobustToolbox\Robust.Shared\Timing\GameLoop.cs:line 200
   at Robust.Client.GameController.MainLoop(DisplayMode mode) in K:\github.com\cyclowns\space-station-14\RobustToolbox\Robust.Client\GameController\GameController.Standalone.cs:line 114
   at Robust.Client.GameController.ParsedMain(CommandLineArgs args, Boolean contentStart, IMainArgs loaderArgs) in K:\github.com\cyclowns\space-station-14\RobustToolbox\Robust.Client\GameController\GameController.Standalone.cs:line 60
   at Robust.Client.GameController.Start(String[] args, Boolean contentStart, IMainArgs loaderArgs) in K:\github.com\cyclowns\space-station-14\RobustToolbox\Robust.Client\GameController\GameController.Standalone.cs:line 35
   at Robust.Client.ContentStart.Start(String[] args) in K:\github.com\cyclowns\space-station-14\RobustToolbox\Robust.Client\ContentStart.cs:line 10
   at Content.Client.Program.Main(String[] args) in K:\github.com\cyclowns\space-station-14\Content.Client\Program.cs:line 9

  This exception was originally thrown at this call stack:
    Content.Shared.Alert.AlertPrototype.GetIcon(short?) in AlertPrototype.cs
    Content.Client.UserInterface.Controls.AlertControl.SetSeverity(short?) in AlertControl.cs
    Content.Client.GameObjects.Components.Mobs.ClientAlertsComponent.UpdateAlertsControls() in ClientAlertsComponent.cs
    Content.Client.GameObjects.Components.Mobs.ClientAlertsComponent.HandleComponentState(Robust.Shared.GameObjects.ComponentState, Robust.Shared.GameObjects.ComponentState) in ClientAlertsComponent.cs
    Robust.Client.GameObjects.ClientEntityManager.HandleEntityState(Robust.Shared.Interfaces.GameObjects.IComponentManager, Robust.Shared.Interfaces.GameObjects.IEntity, Robust.Shared.GameObjects.EntityState, Robust.Shared.GameObjects.EntityState) in ClientEntityManager.cs

Inner Exception 1:
ArgumentException: No severity specified but this alert has severity. (Parameter 'severity')
```

</p>
</details>

maybe caused by 44e3cc93959540c37e1d55cd2436e2f523d68166

**Comment by cyclowns** [comment]
tested after multiple rebuilds, recloning the entire repo, and multiple ways to die so if this is just a fluke on my part then I have no idea

**Comment by metalgearsloth** [comment]
@chairbender

**Comment by cyclowns** [comment]
..appears to be fixed? not sure if it was meant to be fixed or not but i can go into crit now whereas it didn't work before

**Comment by Zumorica** [comment]
Fixed by 74fed841a3886925b27b41d9e8bd8b68c6583ef6
